src/lib.rs

```diff
@@ -163,13 +165,18 @@ impl Module {
     }
 
     /// Returns the method, if it exists in the Module
+    /// Wildcard methods have lower priority than defined methods
     pub fn get_method(&self, method: &str) -> Option<&Method> {
+        let mut outcome: Option<&Method> = None;
         for (name, m) in &self.methods {
             if name == method {
-                return Some(m);
+                outcome = Some(m);
+                break;
+            } else if name == "*" {
+                outcome = Some(m);
             }
```

Comment:
Both the if blocks seem to be doing the same thing: `outcome = Some(m)` <- do we still need it? And we don't need `break` I think

Reply:
^ ignore the above I didn't see the for loop lol